lib/Date/Utility.pm

```diff
@@ -1157,7 +1157,7 @@ sub today {
     return $today_obj;
 }
 
-=head2 _plus_years
+=head2 plus_years
```

Comment:
could we update the pod's based on style guide https://wikijs.deriv.cloud/en/Backend/guides/perl/perl-style-guide#subroutine-comments

Reply:
added pod